psi4/driver/procrouting/proc.py

```diff
@@ -2585,10 +2634,11 @@ def run_scf_hessian(name, **kwargs):
     H = core.scfhess(ref_wfn)
     ref_wfn.set_hessian(H)
 
-    # Clearly, add some logic when the reach of this fn expands
-    ref_wfn.set_variable("HF TOTAL HESSIAN", H)  # P::e SCF
     ref_wfn.set_variable("SCF TOTAL HESSIAN", H)  # P::e SCF
-    core.set_variable("SCF TOTAL HESSIAN", H)  # P::e SCF
+    if ref_wfn.functional().needs_xc():
+        ref_wfn.set_variable("DFT TOTAL HESSIAN", H)  # no DFT hess at present. overwritten later for DH -- TODO when DH gradients  # P::e SCF
```

Comment:
I thought we had LDA hessians.

Reply:
I remember a comment along those lines, but it didn't agree with my memory. We don't test any dft hess. In any case, the code edit will work better with DFT.

Reply:
But we do test the DFT frequencies _from_ an analytic hessian in `dft-freq-analytic`.

The more I look at this like, the more confused I am. What _is_ DH?

Reply:
I had forgotten that. comment edited to `# overwritten later for DH -- TODO when DH Hessians # P::e SCF` to mirror the gradient line above.